backend/README.md

```diff
@@ -0,0 +1,36 @@
+# Eiffelvis Backend
+[![Backend CI](https://github.com/ItJustWorksTM/EiffelVis/actions/workflows/backend_ci.yml/badge.svg)](https://github.com/ItJustWorksTM/EiffelVis/actions/workflows/backend_ci.yml)
+
+Eiffelvis is mainly implemented inside [eiffelvis_core](./crates/eiffelvis_core), it makes up 90% of the actual code along with [eiffelvis_http](./crates/eiffelvis_http). This top level binary simply bundles all these into a usable executable with command line options.
+
+Take a peek inside the `crates` folder if you are curious.
+
+## Basic setup
+
+```c
+# Clone the repo
+git clone "https://github.com/ItJustWorksTM/EiffelVis eiffelvis"
+# Traverse into the backend folder
+cd eiffelvis/backend
+# Run via cargo
+cargo run -- --help
+```
+*Note*: Prebuild binaries are available for all 3 major platforms [here](https://github.com/ItJustWorksTM/EiffelVis/releases), or check the latest ci jobs.
+
+## Usage
+
+Usage is pretty straight forward, there are no config files or settings to modify on your system, eiffelvis will start a web server and connect to the rabbitmq broker with respective given adresses.
+
+```
+-a, --address <address>          HTTP host address [default: 127.0.0.1]
+    --chunk-size <chunk-size>    Maximum amount of events a single chunk will hold [default: 128]
+    --max-chunks <max-chunks>    Maximum amount of chunks stored in memory [default: 8]
```

Comment:
Maybe add something about how the amount has to be a power of 2, with examples, e.g. 2, 4, 8, 16, 32, ...
I know this is already shown in the command line, but you can never trust users.